src/components/Routes/RootRoutes.js

```diff
@@ -60,6 +62,12 @@ export const RootRoutes = () => {
                 <NewAccount/>
             </Route>
 
+            {/* Route for Testing AddItemButton */}
+            <Route path="/AddItemButtonTest">
```

Comment:
For any demo views, let's have it route to `/demo/[componentName]` in this case, this should be `/demo/AddItemButton`